+sig/+node/Signal.m

```diff
@@ -285,28 +285,36 @@
       id.Node.DisplayInputs = this.Node.DisplayInputs;
     end
     
+    function fs = flatten(this)
+      % Returns a signal `fs` whose value is a "flattened" version of 
+      % `this`'s value. `this` is a subscriptable signal field whose value 
+      % is another signal. In other words, `flatten` "flattens" `this`'s 
```

Comment:
This has gone beyond parody.  I defy you find anyone in the lab who thinks this is more readable with quotes.  

Remove the quotes around the word 'flatten' - there is no literal equivalent and therefore no way to misinterpret the meaning. Replace this line with 'If the current value is itself a signal (as is the case with subscriptable signals), the value it holds will be returned, rather than the signal itself.'
https://www-chicagomanualofstyle-org.libproxy.ucl.ac.uk/qanda/data/faq/topics/Punctuation/faq0014.html

Reply:
In that link you posted, there is this: 

> When you decide to use them because they seem like the best way to introduce an unusual phrase or jargon, drop the quotation marks for subsequent uses of the same word or phrase.

I don't think anyone who looks at this function would immediately understand what it means to "flatten" a signal - "flatten" here is not a concise descriptor. I use quotes to try and bring attention to that. That being said, since I've used it once in quotes in a previous sentence, I can remove the quotes the second time.

Reply:
The quotes don't add anything.  How else would someone interpret it?  You can't 3D print a signal then stamp on it underfoot.  Putting quotes around it doesn't add any information to the explanation.  Flatten has no other meaning than the one figurative and intuitive meaning with which it's used here.  If they don't catch the meaning from the function title then re-using it in multiple sentences isn't going to disambiguate anything for them.  It would be similar to writing "Update 'b''s value "at" the time signal 'a''s value updates".  There is a clearer way to write this that doesn't give me a headache haha